frontend/themes/zitate-sammlung/schulmanager.css

```diff
@@ -160,7 +160,7 @@ span.origi, span.origi > b {
 
 .timetable-nav {
     position: fixed;
-    bottom: 60px;
+    bottom: 74px;
```

Comment:
❌ Change to 78px, as the padding you removed is 18px. Otherwise, timetable-nav's shadow floats in front of the navbar.

Reply:
i don't see any shadow but alright

Reply:
![image](https://github.com/IllusioQuest/teacher-quotation-manager/assets/143633974/87dcaec5-2edd-44d4-a70a-42966e3c6513)
Overlapping area's small, but you see what I mean?